packages/x-charts/src/ChartsTooltip/ChartsTooltipTable.ts

```diff
@@ -2,6 +2,9 @@ import { styled } from '@mui/material/styles';
 import { shouldForwardProp } from '@mui/system';
 import { chartsTooltipClasses } from './chartsTooltipClasses';
 
+/**
+ * @ignore - internal component.
+ */
 export const ChartsTooltipPaper = styled('div', {
   name: 'MuiChartsTooltip',
```

Comment:
I find it strange to have this component as ignore (` * @ignore - internal component.`) and having a name that is reachable with theme overrides (and normally global CSS too, I mean, I imagine Pigment CSS output clear class names in this context using the name of the styled called, assuming uniqueness property is guaranteed cc @brijeshb42 for context).